srcpkgs/openvpn/template

```diff
@@ -1,20 +1,19 @@
 # Template file for 'openvpn'
 pkgname=openvpn
-version=2.4.9
-revision=3
+version=2.5.0
+revision=1
 build_style=gnu-configure
-configure_args="$(vopt_enable pkcs11) --enable-iproute2 --disable-systemd
```

Comment:
What was iproute2 dropped in this update?

Reply:
Iproute2 is note required any more to manage the routing tables. OpenVPN from 2.5.0 has Netlink integaration. See https://openvpn.net/community-downloads-2/ for details.